src/models/eventsFactory.js

```diff
@@ -314,12 +334,53 @@ class EventsFactory extends Factory {
 
     for (let i = 0; i < days; i++) {
       const now = new Date();
-      const day = new Date(now.setDate(now.getDate() - i));
-      const dayMidnight = getUTCMidnight(day) / 1000;
-      const groupedEvents = groupedData[`groupingTimestamp:${dayMidnight}`];
+
+      now.setHours(0, 0, 0, 0);
```

Comment:
why to not use `getUTCMidnight` instead?

Reply:
`getUTCMidnight` sounds like a pure function but in fact it implicitly change passed date. So I think explicit set is better here

Reply:
so you can rename it with `setUTCMidnight`. Using a separate method will improve the readability of this code